wren-ai-service/src/pipelines/generation/followup_sql_generation.py

```diff
@@ -20,7 +20,8 @@
     sql_generation_system_prompt,
 )
 from src.utils import async_timer, timer
-from src.web.v1.services.ask import AskConfigurations, AskHistory
+from src.web.v1.services import Configuration
+from src.web.v1.services.ask import AskHistory
```

Comment:
nitpick: we could consider moving them into pipeline layer in the future. i think it can reduce the bi-dependency between pipeline and service layer